lib/semantics/check-declarations.cc

```diff
@@ -17,6 +17,7 @@
 #include "../evaluate/check-expression.h"
 #include "../evaluate/fold.h"
 #include "../evaluate/tools.h"
+#include <algorithm>
```

Comment:
Is this needed?

Reply:
Not any more; thanks.  I was using `std::find_if()` for a while.